inc/admin/customizer.php

```diff
@@ -272,6 +281,7 @@ function alcatraz_customize_register( $wp_customize ) {
 			'default'    => '',
 			'type'       => 'option',
 			'capability' => 'edit_theme_options',
+			'sanitize_callback' => 'sanitize_text_field',
```

Comment:
Also, using sanitize_text_field for our footer_bottom option will prevent us from using HTML there. I think wp_kses_post might be a better way to go.